clang/labs/lab1/korablev_nikita/RenameIdentificator.cpp

```diff
@@ -0,0 +1,146 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+#include "clang/Rewrite/Core/Rewriter.h"
+
+#include "llvm/Support/raw_ostream.h"
+#include "llvm/ADT/StringRef.h"
+
+#include "clang/ASTMatchers/ASTMatchFinder.h"
+#include "clang/ASTMatchers/ASTMatchers.h"
+#include "clang/Tooling/Transformer/RewriteRule.h"
+#include "clang/Tooling/Transformer/Transformer.h"
+#include "clang/Tooling/Transformer/Stencil.h"
+
+using namespace clang;
+using namespace clang::ast_matchers;
+using namespace clang::transformer;
```

Comment:
Please, avoid using "using namespace *". "Using" statement for particular namespace members is ok (like ast_matchers and transformer)